notebooks/enterprise-search/app-search-engine-exporter.ipynb

```diff
@@ -0,0 +1,894 @@
+        "          \"lowercase\",\n",
+        "          \"asciifolding\",\n",
+        "          \"en-stop-words-filter\",\n",
+        "          \"en-stem-filter\",\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      },\n",
+        "      \"i_text_bigram\": {\n",
+        "        \"filter\": [\n",
+        "          \"cjk_width\",\n",
+        "          \"lowercase\",\n",
+        "          \"asciifolding\",\n",
+        "          \"en-stem-filter\",\n",
+        "          \"bigram_joiner\",\n",
+        "          \"bigram_max_size\"\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      },\n",
+        "      \"q_text_bigram\": {\n",
+        "        \"filter\": [\n",
+        "          \"cjk_width\",\n",
+        "          \"lowercase\",\n",
+        "          \"asciifolding\",\n",
+        "          \"synonyms-filter\",\n",
+        "          \"en-stem-filter\",\n",
+        "          \"bigram_joiner_unigrams\",\n",
+        "          \"bigram_max_size\"\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      }\n",
+        "    }\n",
+        "  }\n",
+        "}\n",
+        "\n",
+        "# and actually create our index\n",
+        "elasticsearch.indices.create(\n",
+        "  index= DEST_INDEX,\n",
+        "  mappings={\n",
+        "    \"properties\": mapping\n",
+        "  },\n",
+        "  settings=settings\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Add `sparse_vector` fields for semantic search\n",
+        "\n",
+        "One of the advantages of having our exported index directly in Elasticsearch is that we can easily take advantage of doing semantic search with ELSER.\n",
+        "Let's first start by adding `sparse_vector` fields to our new index mapping."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# by default we are adding a `sparse_vector` field for all text fields in our engine\n",
+        "# feel free to modify this list to only include the fields that are relevant\n",
+        "SPARSE_VECTOR_FIELDS = [field_name + \"_semantic\" for field_name in schema if schema[field_name] == \"text\"]\n",
+        "\n",
+        "sparse_vector_fields = {}\n",
+        "for field_name in SPARSE_VECTOR_FIELDS:\n",
+        "  # this is added so we can use semantic search with ELSER\n",
+        "  sparse_vector_fields[field_name] = {\"type\": \"sparse_vector\"}\n",
+        "\n",
+        "elasticsearch.indices.put_mapping(\n",
+        "  index=DEST_INDEX,\n",
+        "  properties=sparse_vector_fields\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Setup an ingest pipeline using ELSER\n",
```

Comment:
We could do this via API commands too, but I think this is fine

Reply:
I'll keep this here for now to keep things simple, but will point the user in the direction if they want to read / use the API instead.